code/modules/clothing/suits/armor.dm

```diff
@@ -130,7 +130,7 @@
 		rad = 0
 	)
 	price_tag = 600
-	slowdown = MEDIUM_SLOWDOWN
+	slowdown = 0.1
```

Comment:
This means the armor will need to be updated manually if slowdown is ever changed.
Should be light_slowdown instead.